themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1910,25 +1921,62 @@ a.tc-tiddlylink.tc-plugin-info:hover .tc-plugin-info > .tc-plugin-info-chunk > s
 */
 
 .tc-chooser {
-	border: 1px solid <<colour table-border>>;
+	border-right: 1px solid <<colour table-header-background>>;
+	border-left: 1px solid <<colour table-header-background>>;
 }
 
+
 .tc-chooser-item {
-	border: 8px;
-	padding: 2px 4px;
+	border-bottom: 1px solid <<colour table-header-background>>;
+	border-top: 1px solid <<colour table-header-background>>;
+	padding: 2px 4px 2px 14px;
+}
+
+.tc-drop-down .tc-chooser-item {
+	padding: 2px;
+}
+
+.tc-chosen,
+.tc-chooser-item:hover {
+	background-color: <<colour table-header-background>>;
+	border-color: <<colour table-footer-background>>;
+}
```

Comment:
@pmario, currently, we have no hover highlight for table borders or headers, e.g. hover states in dropdowns. The problem with using more contrasty backgrounds for the selected option in a sidebar popup is that, since link colours are grey'ish, the contrast between the text and the background will suffer if we use a darker background for the selected option. However, I deliberately did not use the inverted colours, especially with regards to the paletteswitcher.

The choosers are **not** exactly dropdowns, although used in such via PageControl popups. Those in the control center are always visible right away, and basically more like table-like lists, hence my inclination to use table styles.

When used in drop-downs, I think their "link" styles should not be overruled / fully inverted like we do with basic dropdowns, at least that doesn't work well with the language switcher and especially not the palette. Hence, my decision to not use this type of "dramatic" highlighting at all.

If tables styles end up producing very little contrast for switchers then so they do for tables. I believe, in that case, that is what the theme designer wanted.

I'm also quite fine with using the table header background for the highlighted item so as to give it a more prominent, yet subtle highlight.

In other words, I'm quite fine with the kind of gentle highlight they have atm. If you wish to play with it more, perhaps upload a demo with proposed other colors somewhere? However, I would recommend to only address dropdown related styles or to address them separately.

What I wanted to avoid was to arbitrarily pick different colours from different aspects of TiddlyWiki. So, having them all be table-based works well in that respect. But perhaps yes, the dropdown colours could be of a different origin than those used in the control center.